.husky/pre-commit

```diff
@@ -2,3 +2,15 @@
 . "$(dirname "$0")/_/husky.sh"
 
 npx lint-staged
+
+declare -a THEMES
+DATE=$(date)
+THEMES=($(git diff --cached --name-only | grep -i "theme" || :))
```

Comment:
Do we need to include the styles path somehow? Could this bite us if we ever made a file like `themeUtils.ts`?

Reply:
I tested it out, and it time-stamped my TS file 😆

Reply:
🤦🏻